[2025-05-24T00:19:44.765+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cancellations_etl_dag.transform_data scheduled__2025-05-23T00:00:00+00:00 [queued]>
[2025-05-24T00:19:44.838+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cancellations_etl_dag.transform_data scheduled__2025-05-23T00:00:00+00:00 [queued]>
[2025-05-24T00:19:44.838+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2025-05-24T00:19:44.845+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): transform_data> on 2025-05-23 00:00:00+00:00
[2025-05-24T00:19:44.849+0000] {standard_task_runner.py:57} INFO - Started process 211 to run task
[2025-05-24T00:19:44.852+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'cancellations_etl_dag', 'transform_data', 'scheduled__2025-05-23T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/cancellations_etl_dag.py', '--cfg-path', '/tmp/tmp_7j_hpru']
[2025-05-24T00:19:44.854+0000] {standard_task_runner.py:85} INFO - Job 4: Subtask transform_data
[2025-05-24T00:19:44.908+0000] {task_command.py:415} INFO - Running <TaskInstance: cancellations_etl_dag.transform_data scheduled__2025-05-23T00:00:00+00:00 [running]> on host a5551f46ca35
[2025-05-24T00:19:44.979+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='cancellations_etl_dag' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-23T00:00:00+00:00'
[2025-05-24T00:19:44.997+0000] {logging_mixin.py:151} INFO - Columns in cancellation data: Index(['Textbox22', 'Textbox24', 'financial_period_key',
       'National_or_Operator', 'Trains_planned', 'Trains_part_cancelled1',
       'Trains_full_cancelled', 'Cancellation_score__number_',
       'NR_Responsible_infrastructure', 'NR_responsible_external_Event',
       'TOC_responsible_TOC_fault', 'TOC_responsible_external_event',
       'Percent_cancelled_periodic', 'Percent_cancelled_periodic__MAA_'],
      dtype='object')
[2025-05-24T00:19:45.011+0000] {logging_mixin.py:151} INFO -                                        Textbox22  ... Percent_cancelled_periodic__MAA_
0  Periodic cancellations score \r\n(percentage)  ...                              [z]
1  Periodic cancellations score \r\n(percentage)  ...                              [z]
2  Periodic cancellations score \r\n(percentage)  ...                              [z]
3  Periodic cancellations score \r\n(percentage)  ...                              [z]
4  Periodic cancellations score \r\n(percentage)  ...                              [z]

[5 rows x 14 columns]
[2025-05-24T00:19:45.025+0000] {logging_mixin.py:151} INFO - Cleaned cancellation data saved to data/processed/cancellation_cleaned.csv
[2025-05-24T00:19:45.025+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-24T00:19:45.032+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=cancellations_etl_dag, task_id=transform_data, execution_date=20250523T000000, start_date=20250524T001944, end_date=20250524T001945
[2025-05-24T00:19:45.073+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-24T00:19:45.087+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
